Index: app/src/main/java/com/iflytek/test/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/iflytek/test/MainActivity.java	(revision 0209da58a3892fd530bb3cf2b471f702b950308c)
+++ app/src/main/java/com/iflytek/test/MainActivity.java	(revision d9de58448fbf75965671110cfa7aa60ee03cfd4c)
@@ -15,5 +15,6 @@
 
     @Override
     public void onClick(View v) {
+        //测试提交
     }
 }
